plugins/modules/k8s_info.py

```diff
@@ -142,58 +142,50 @@
 
 import copy
 
-from ansible.module_utils.basic import AnsibleModule
-from ansible_collections.community.kubernetes.plugins.module_utils.common import (
-    K8sAnsibleMixin, AUTH_ARG_SPEC, WAIT_ARG_SPEC)
-
-
-class KubernetesInfoModule(K8sAnsibleMixin):
-
-    def __init__(self, *args, **kwargs):
-        module = AnsibleModule(
-            argument_spec=self.argspec,
-            supports_check_mode=True,
+from ansible_collections.cloud.common.plugins.module_utils.turbo.module import (
```

Comment:
I would like to see a toggle to change behavior.

Reply:
At this stage in the module loading, we cannot access the module parameters yet. As an alternative, I can use an environment variable or just fallback on AnsibleModule in case of ImportError.